cogboard-webapp/src/components/UserLogin/index.js

```diff
@@ -53,26 +53,30 @@ const UserLogin = () => {
   return (
     <>
       {!isAuthenticated && (
-        <IconButton
-          onClick={handleLoginDialogOpen}
-          aria-label="Login"
-          color="inherit"
-          edge="start"
-          data-cy="user-login-login-icon"
-        >
-          <AccountCircle color="primary" />
-        </IconButton>
+        <Tooltip title="Login" placement="bottom-end">
+          <IconButton
+            onClick={handleLoginDialogOpen}
+            aria-label="Login"
+            color="primary"
+            edge="start"
+            data-cy="user-login-login-icon"
+          >
+            <AccountCircle color="primary" />
+          </IconButton>
+        </Tooltip>
       )}
       {isAuthenticated && (
-        <IconButton
-          onClick={handleLogout}
-          aria-label="Logout"
-          color="inherit"
-          edge="start"
-          data-cy="user-login-logout-icon"
-        >
-          <PowerSettingsNew color="primary" />
-        </IconButton>
+        <Tooltip title="Logout" placement="bottom-end">
+          <IconButton
+            onClick={handleLogout}
+            aria-label="Logout"
+            color="primary"
+            edge="start"
+            data-cy="user-login-logout-icon"
+          >
+            <PowerSettingsNew color="error" />
```

Comment:
shouldn't the color change only on mouse hoover

Reply:
Yeah I change it